tests/data_inputs/count_based.json

```
@@ -0,0 +1,27 @@
+[
+    {
+        "count": 10,
+        "month": "2025-01",
+        "county": "A",
+        "zip_code": "12345",
+        "zip_code_population": 3000,
+        "purpose": "Provide 1 small cell that will force latent suppression."
+    },
+    {
+        "count": 20,
+        "month": "2025-01",
+        "county": "A",
+        "zip_code": "23456",
+        "zip_code_population": 4000,
+        "demographics": "Profile-A",
+        "purpose": "Be latently suppressed."
+    },
+    {
+        "count": 21,
+        "month": "2025-01",
+        "county": "B",
+        "zip_code": "23456",
+        "zip_code_population": 4000,
+        "purpose": "Be displayed, not suppressed."
+    }
```

I'm wondering if this test input could be a bit simpler? I'm not sure if I'm missing something but since the thrust seems to be asserting that a summative "whole" is produced from component parts for the purposes of latent-revelation checking, my suggestion is to:

* Remove the demographic profile associated with the second record
* Update the third record to be a simpler "separate geography" (ZIP code)
  * As it stands, it looks more like an example of a geographic sliver problem 

```suggestion
    {
        "count": 20,
        "month": "2025-01",
        "county": "A",
        "zip_code": "23456",
        "zip_code_population": 4000,
        "purpose": "Be latently suppressed."
    },
    {
        "count": 21,
        "month": "2025-01",
        "county": "A",
        "zip_code": "34567",
        "zip_code_population": 4000,
        "purpose": "Be displayed, not suppressed."
    }
```

---

I have another PR following up on this where I'll remove the demographic profile. 

I actually wanted the sliver based on the tests I was running before, I think the expected output is the same but we can discuss further if you think this is confusing or inaccurate.